website/src/pages/carrot_etl_workflow.mdx

```diff
@@ -0,0 +1,461 @@
+
+**JVM (Java Virtual Machine)**: The software that runs WhiteRabbit. Think of it as the "engine" that powers the tool.
+
+**PyPI**: The Python Package Index - a website where you can download Python tools like Carrot Transform.
+
+**CSV (Comma-Separated Values)**: A simple file format where data is separated by commas, like: "Name,Age,City" on one line, then "John,25,London" on the next line.
+
+**SAS**: A file format used by SAS statistical software, commonly used in healthcare and research.
+
+---
+
+## Phase 1: Data Profiling with WhiteRabbit
+
+### Step 1: Installation and Setup
+
+#### 1.1 Download WhiteRabbit
+1. Go to [OHDSI WhiteRabbit releases](https://github.com/OHDSI/WhiteRabbit/releases/latest)
+2. Download the latest `WhiteRabbit_vX.X.X.zip` file
+3. Extract the zip file to your desired location
+
+#### 1.2 Verify Installation
+Check that you have the following structure:
+```
+WhiteRabbit_v1.0.0/
+├── bin/
+│   ├── whiteRabbit (macOS/Linux)
+│   ├── whiteRabbit.bat (Windows)
+│   ├── rabbitInAHat (macOS/Linux)
+│   └── rabbitInAHat.bat (Windows)
+├── repo/ (contains JAR files)
+└── examples/ (contains sample data)
+```
+
+#### 1.3 Launch WhiteRabbit
+**On macOS/Linux:**
+```bash
+cd /path/to/WhiteRabbit_v1.0.0
+./bin/whiteRabbit
+```
+
+**On Windows:**
+```cmd
+cd C:\path\to\WhiteRabbit_v1.0.0
+bin\whiteRabbit.bat
+```
+
+**Expected Result:** WhiteRabbit GUI opens with configuration sections.
+
+### Step 2: Data Source Configuration
+
+#### 2.1 Set Working Folder
+1. In WhiteRabbit GUI, locate the "Working Folder" section
+2. Click "Pick Folder" button
+3. Navigate to the folder containing your source data files
+4. Select the folder and confirm
+
+#### 2.2 Configure Data Source Type
+**For CSV/Text Files:**
+1. Select "Delimited text files" as source type
+2. Set delimiter:
+   - `comma` for CSV files
+   - `tab` for TSV files
+   - `semicolon` for semicolon-delimited
+   - `pipe` for pipe-delimited
+
+**For SAS Files:**
+1. Select "SAS" as source type
+2. No additional configuration needed
+
+**For Database:**
+1. Select your database type (MySQL, Oracle, PostgreSQL, SQL Server, etc.)
+2. Enter connection details:
+   - Server location (host:port)
+   - Username and password
+   - Database name
+3. Click "Test connection" to verify
+
+#### 2.3 Configure Scan Options
+1. **Scan field values**: Check this box (recommended)
+2. **Min cell count**: Set to 5 (default) - minimum frequency for values to appear
+3. **Rows per table**: Choose sampling size:
+   - 100,000 (default, good for most cases)
+   - 500,000 (for larger datasets)
+   - 1,000,000 (for very large datasets)
+   - All rows (for complete analysis)
+4. **Max distinct values**: Set to 1,000 (default) - maximum unique values per field
+5. **Numeric stats**: Check if you want statistical analysis of numeric fields
+
+### Step 3: Execute Data Scan
+
+#### 3.1 Select Files/Tables to Scan
+1. **For file-based sources**: Select specific files from the working folder
```

Comment:
Some images to support these would be helpful, not for every step but at least in key parts.